.github/workflows/plugin_test.yml

```diff
@@ -0,0 +1,22 @@
+name: Plugin Test
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Does the default CI image have Docker installer? CMIIW but it is needed for running dockertest.

Reply:
Although the test were running fine in CI, let me verify it again